libnavigation-ui/src/main/java/com/mapbox/navigation/ui/map/MapWayName.java

```diff
@@ -50,7 +50,7 @@ void updateWayNameWithPoint(PointF point) {
   }
 
   void updateProgress(List<Point> currentStepPoints) {
-    if (!this.currentStepPoints.equals(currentStepPoints)) {
+    if (currentStepPoints != null && !this.currentStepPoints.equals(currentStepPoints)) {
```

Comment:
How's that working in the legacy 🤔 https://github.com/mapbox/mapbox-navigation-android/blob/41b9a2e293bad7aa3cc82b19d03e523d8c2caf9c/libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/map/MapWayName.java#L53